GMAO_pFIO/CMakeLists.txt

```diff
@@ -99,11 +99,9 @@ set (srcs
   )
 
 esma_add_library (${this} SRCS ${srcs})
-target_link_libraries (${this} gftl ${NETCDF_LIBRARIES})
+target_link_libraries (${this} PUBLIC Baselibs)
 
-target_compile_options (${this} PRIVATE $<$<COMPILE_LANGUAGE:Fortran>:${OpenMP_Fortran_FLAGS}>)
 set_target_properties (${this} PROPERTIES Fortran_MODULE_DIRECTORY ${include_${this}})
-target_link_libraries (${this} ${MPI_Fortran_LIBRARIES} ${OpenMP_Fortran_LIBRARIES})
```

Comment:
@LiamBindle Thanks - I'd always wondered why we were needing to repeat this logic here.   I'm too Fortran-centric to have noticed that Fortran is only an option for the FindOpenMP() flag.

Not going to approve this yet, as it is contingent on the other PR.